juniper_official/Protocols/Ospf/ospf-neighbor-state.rule

```diff
@@ -70,4 +70,4 @@ iceberg {
             }
```

Comment:
410
Entering configuration mode
The configuration has been changed but not committed
|/rules/juniper_official/Protocols/Ospf/ospf-neighbor-state.rule:72:(0) syntax error
load complete (1 errors)

Reply:
Addressed and resolved